src/components/atoms/SpacesDropdown/SpacesDropdown.tsx

```diff
@@ -60,7 +60,7 @@ export const SpacesDropdown: React.FC<SpacesDropdownProps> = ({
 
         return (
           <ReactBootstrapDropdown.Item
-            key={id}
+            key={id + name}
```

Comment:
`id` is unique. It shouldn't need `name` adding to it.
```suggestion
            key={id}
```

Reply:
Looks like it isn't. I don't know why but React warning was showing up for duplicated keys.